umn/source/faqs/how_to_access_kibana_from_elb.rst

```diff
@@ -0,0 +1,163 @@
+.. _css_02_0201:
+
+How To Access Kibana From ELB
```

Comment:
I would consider renaming the chapter name to something like:
How to Access Kibana from outside of the cloud using ELB

Reply:
Thanks for suggesting new heading for thie page, i have updated it :)